docs/v1/P4Runtime-Spec.mdk

```diff
@@ -4695,6 +4696,10 @@ message PacketMetadata {
 
 ## Master Arbitration Update
 
+P4Runtime's master arbitration mechanism ensures the following invariant:
+the device will not allow a write from an older master, if it has already
+accepted a write from a newer master.
+
```

Comment:
Should this be changed to "if it has already accepted a `MasterArbitrationUpdate` message from a newer master."?  Otherwise, it seems the mastership change is enforced only after the new master sends a `WriteRequest`, thus allowing an old master to modify state after a mastership change,  as defined in the rest the spec, has already occurred.

Reply:
Great point, done!